Scala/Hangman.scala

```diff
@@ -0,0 +1,294 @@
+              }
+
+
+              println("\nYou have used the following letters: " + usedLetters)
+
+              println("\nPlease choose a letter.\n")
+              var input: String = scala.io.StdIn.readLine()
+
+              usedLetters += input
+
+              theWord.toCharArray.contains(input.charAt(0)) match {
+                case false => hangCount += 1
+                case true =>
+              }
+
+              hangCount match {
+                case 0 =>
+                  println("------")
+                  println("|  |")
+                  println("| (_)")
+                  println("|")
+                  println("|")
+                  println("|_____")
+
+                case 1 =>
+                  println("------")
+                  println("|  |")
+                  println("|  O")
+                  println("|")
+                  println("|")
+                  println("|_____")
+
+                case 2 =>
+                  println("------")
+                  println("|  |")
+                  println("|  O")
+                  println("|  |")
+                  println("|")
+                  println("|_____")
+
+                case 3 =>
+                  println("------")
+                  println("|  |")
+                  println("|  O")
+                  println("| /|")
+                  println("|")
+                  println("|_____")
+
+                case 4 =>
+                  println("------")
+                  println("|  |")
+                  println("|  O")
+                  println("| /|\\")
+                  println("|")
+                  println("|_____")
+
+                case 5 =>
+                  println("------")
+                  println("|  |")
+                  println("|  O")
+                  println("| /|\\")
+                  println("| / ")
+                  println("|_____")
+
+                case 6 =>
+                  println("------")
+                  println("|  |")
+                  println("|  O")
+                  println("| /|\\")
+                  println("| / \\")
+                  println("|_____")
+
+              }
+          }
+      }
+    }
+
+  }
+}
+
+def DBUpdate(inQ:String) =
+{
+// Connects Scala Object 'Main' to the SQL Database.
+
+val url = "jdbc:mysql://192.168.1.49:3306/hangmanDB"
+val driver = "com.mysql.jdbc.Driver"
+val username = "root"
+val password = "merdegraw494"
+var connection:Connection = DriverManager.getConnection(url, username, password)
+try {
+Class.forName(driver)
+
+connection = DriverManager.getConnection(url, username, password)
+val statement = connection.createStatement
+
+// Connection strings go here
+statement.executeUpdate(inQ)
+
+} catch {
+case e: Exception => e.printStackTrace
+}
+connection.close
+}
+
+def DBRead(inQ:String):String =
+{
+// Connects Scala Object 'Main' to the SQL Database.
+
+val url = "jdbc:mysql://192.168.1.49:3306/hangmanDB"
+val driver = "com.mysql.jdbc.Driver"

```

Comment:
Same connection values duplicated, could have them in an enum file and grab them from there, making it cleaner.  Or possibly create a trait which would have a method that would return the connection to the database.